lemur/auth/views.py

```diff
@@ -184,8 +185,6 @@ def create_user_roles(profile):
                 current_app.config["LEMUR_DEFAULT_ROLE"],
                 description="This is the default Lemur role.",
             )
-        if not default.third_party:
```

Comment:
are we removing this part?

Reply:
Added it back since it may have side-effect of adding default role with each login